NEWS.md

```diff
@@ -4,6 +4,7 @@ This update contains a substantial package refactor and as such introduces some
 
 ## Changes
 
+* Placed nhs data handling in correct areas of UK class, so that downloading occurs at the download step, cleaning happens at the cleaning step, etc. Also added tests.
```

Comment:
Sorry I know I said do this but only for changes that impact the user (or in a new section along the lines of package updates but not in this instance as in this version the whole package will change!)

Reply:
I see, got it. will revert this